test/mineflayer.test.js

```diff
@@ -243,11 +243,12 @@ squid.supportedVersions.forEach((supportedVersion, i) => {
         await waitMessage(bot, 'You do not have permission to use this command')
         serv.getPlayer('bot').op = true
       })
-      test('can use /setblock', async () => {
+      test('can use /setblock', async (done) => {
         await Promise.all([waitSpawnZone(bot, 2), onGround(bot)])
         bot.chat('/setblock 1 2 3 95 0')
         let [, newBlock] = await once(bot, 'blockUpdate:' + new Vec3(1, 2, 3), { array: true })
         expect(newBlock.type).toEqual(95)
+        done()
```

Comment:
I don't know why this part always throws an error which display it time out. I don't think this part have any problem before.